botocore/loaders.py

```diff
@@ -473,6 +483,23 @@ def _potential_locations(self, name=None, must_exist=False, is_dir=False):
                     elif os.path.exists(full_path):
                         yield full_path
 
+    def is_builtin_path(self, path):
+        """Whether a given path is within the package's data directory
+
+        This method can be used together with load_data(name, include_path=True)
+        to determine if data has been loaded from a file bundled with the
+        package, as opposed to a file in a separate location.
+
+        :type path: str
+        :param path: The file path to check.
+
+        :return: Whether the given path is within the package's data directory.
+
+        """
+        # normalize and absolutize the path, self.BUILTIN_DATA_PATH already is
+        path = os.path.realpath(path)
```

Comment:
should we be using `abspath` instead here?

Reply:
I'm not going to remove `realpath()` because resolving symlinks is the primary thing I want from this function. But I do now realize that my inline comment sounds a lot more like `abspath()` and that won't hurt either so now it's both.